post_installation_configuration/configuring-multi-arch-compute-machines/multi-architecture-configuration.adoc

```diff
@@ -22,21 +22,75 @@ For information on migrating your single-architecture cluster to a cluster that
+To create a cluster with multi-architecture compute machines with different installation options and platforms, you can use the documentation in the following table:
 
-* xref:../../post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-azure.adoc#creating-multi-arch-compute-nodes-azure[Creating a cluster with multi-architecture compute machines on Azure]
+.Cluster with multi-architecture compute machine installation options
+[cols="3,1,1,1,1,1",options="header"]
+|===
+|Documentation section |Platform |User-provisioned installation |Installer-provisioned installation |Control Plane |Compute node
 
-* xref:../../post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-aws.adoc#creating-multi-arch-compute-nodes-aws[Creating a cluster with multi-architecture compute machines on AWS]
+|xref:../../post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-azure.adoc#creating-multi-arch-compute-nodes-azure[Creating a cluster with multi-architecture compute machines on Azure]
+|Microsoft Azure
+|
+|&#10003;
+|`x86_64`
+|`ARM64`
 
-* xref:../../post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-gcp.adoc#creating-multi-arch-compute-nodes-gcp[Creating a cluster with multi-architecture compute machines on GCP]
+|xref:../../post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-aws.adoc#creating-multi-arch-compute-nodes-aws[Creating a cluster with multi-architecture compute machines on AWS]
+|Amazon Web Services (AWS)
+|
+|&#10003;
+|`x86_64`
+|`ARM64`
 
-* xref:../../post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-bare-metal.adoc#creating-multi-arch-compute-nodes-bare-metal[Creating a cluster with multi-architecture compute machines on bare metal]
+|xref:../../post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-gcp.adoc#creating-multi-arch-compute-nodes-gcp[Creating a cluster with multi-architecture compute machines on GCP]
+|Google Cloud Platform (GCP)
+|
+|&#10003;
+|`x86_64`
+|`ARM64`
 
-* xref:../../post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-z.adoc#creating-multi-arch-compute-nodes-ibm-z[Creating a cluster with multi-architecture compute machines on {ibm-z-name} and {ibm-linuxone-name} with z/VM]
+.3+|xref:../../post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-bare-metal.adoc#creating-multi-arch-compute-nodes-bare-metal[Creating a cluster with multi-architecture compute machines on bare metal, {ibm-power-title}, or {ibm-z-title}]
+|Bare metal
```

Comment:
the title here says baremetal and z/P but the combination here lists x86_64 and ARM..maybe this should be split or called out?

Reply:
i think it's just enough to include arm64 in the description as well